documentation/airport-guides/omdb.md

```diff
@@ -0,0 +1,137 @@
+
+### Standard Operating Procedures
+###### Preferential Runway
+As mentioned previously, Dubai is the world's busiest dual runway airport in terms of traffic. It is common to see one runway designated to departures, and one designated to arrivals. However, it is not uncommon for a few aircraft to land on the departure runway.
+
+When parallel runway operations are in use, one runway should be used for departures, the other for arrivals.
+
+When winds favour the operation of runways 12L/R:
+- 12L for landing
+- 12R for takeoff
+
+When winds favour the operation of runways 30L/R:
+- 30L for landing
+- 30R for takeoff
+
+In OpenScope, a 30 configuration is simulated, with aircraft automatically being assigned runways according to their flight type (departure/arrival). A few arriving aircraft will be automatically assigned 30R.
+
+###### Low Visibility Operations
+Low Visibility Operations are enforced when any of the following criteria is met:
+- Touchdown RVR readings indicate a visibility of 600m or below
+- METAR visibility indicates visibility of 600m or less
+- Reported cloud ceiling is less than 300ft.
+
+In this situation, runway 30L/12R are used for landing and 30R/12L are used for takeoff, with the ILS for the departing runway disabled.
+
+##### Speed Restrictions
+According to the AIP, "ATC has zero tolerance with regards to the compliance with speed control instructions. Pilots shall adhere to the speed (IAS) approved or assigned by ATC and shall request ATC approval before making any changes thereto".
+
+Air Traffic Control would normally issue the following speed control restrictions:
+* 210 – 250 KT: from CTA entry to downwind;
+* 180 – 230 KT: from downwind to base leg;
+* 160 – 210 KT: on base leg and closing heading to final approach;
+* 180 KT: 10 NM from touchdown;
+* 160 KT: 4 NM from touchdown
+
+All speed restrictions on the STAR coding tables will be followed by the pilot automatically. These can only be bypassed with permission.
+
+For departures, the restriction is 250KT below 10,000ft.
+
+##### Reduced Wake Turbulence Separation for Parallel Approaches
+|     Leading Aircraft 30L     | Trailing Aircraft 30R | Minimum Separation |
+| any wake turbulance category |   Super Heavy/Heavy   |   None required    |
+|            Heavy             |         Medium        |     2.5nm/3nm      |
+|         Super Heavy          |         Medium        |       4.5nm        |
+|            Medium            |         Medium        |       2.5nm        |
+
+### Instrument Procedures
+Dubai's instrument procedures all require aircraft to have RNAV capabilities. Furthermore, Dubai's procedures require almost little input from the controller; climb and descent instructions must be issued by the controller. Vectors are rarely used, as all STARs end on the extended centrelines, where an approach clearance (ie. ILS approach) can be issued.

```

Comment:
I would perhaps add that shortcuts are often given on the downwind when possible.